src/course-unit/data/thunk.js

```diff
@@ -113,6 +115,31 @@ export function editCourseItemQuery(itemId, displayName, sequenceId) {
   };
 }
 
+export function editCourseUnitVisibilityAndData(itemId, type, isVisible) {
+  return async (dispatch) => {
+    dispatch(updateSavingStatus({ status: RequestStatus.PENDING }));
+    dispatch(updateQueryPendingStatus(true));
+    const notificationMessage = getNotificationMessage(type, isVisible);
+    dispatch(showProcessingNotification(notificationMessage));
+
+    try {
+      await handleCourseUnitVisibilityAndData(itemId, type, isVisible).then(async (result) => {
```

Comment:
Is it transferred? I would prefer to use `await`. `await` is the same as `.then(`
```suggestion
      const result = await handleCourseUnitVisibilityAndData(itemId, type, isVisible);
      ...
```

Reply:
We use this approach in all the thunks on this page, let's not stand out from the crowd)